site/reference/live/install-resource-group/README.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "`install-resource-group`"
+linkTitle: "install-resource-group"
+type: docs
+description: >
+Install the ResourceGroup CRD in the cluster.
+---
+<!--mdtogo:Short
+    Install the ResourceGroup CRD in the cluster.
+-->
+
+`install-resource-group` installs the ResourceGroup CRD in the cluster. `kpt`
```

Comment:
explain why/when you should run this command?

Reply:
Changed the language a bit here. Ideally we should link to the book here, since a thorough explanation of why this is needed becomes a description of the overall design for `kpt live`.